### PR TITLE
fix: add `appended` property to `drag` event

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* added `appended` property to `drag` event

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jung-han — APPROVED** [review]
완료입니다!

File: packages/toast-ui.grid/src/view/bodyArea.tsx [hunk 1]
```diff
@@ -162,23 +162,29 @@ class BodyAreaComp extends Component<Props> {
...
+      } else {
+        // move the row to next index
+        this.movedIndexInfo = { index, rowKey: rowKeyToMove };
+        this.props.dispatch('moveRow', rowKey, index);
+      }
+
+      const gridEvent = new GridEvent({
+        rowKey,
+        targetRowKey: rowKeyToMove,
+        appended: !!this.movedIndexInfo!.appended,
```
Comment (lja1018) [inline packages/toast-ui.grid/src/view/bodyArea.tsx:176]: `!!` 없어도 되지 않나요?

**Review by lja1018 — APPROVED** [review]
리뷰완료합니다.

**Review by jungeun-cho — APPROVED** [review]
이견없습니다.

**Pull request merged**